python/main.py

```diff
@@ -18,14 +26,87 @@
     allow_headers=["*"],
 )
 
+#For saving into JSON file
+'''
+def write_json(new_item, filename='items.json'):
+    open(filename, 'a').close()
+    if os.stat(filename).st_size == 0:
+        with open(filename, 'w') as fp:
+            res = {"items":[new_item]}
+            json.dump(res,fp)
+    else:
+        with open(filename,'r+') as fp:
+            file_data = json.load(fp)
+            # Join new_data with file_data inside emp_details
+            file_data["items"].append(new_item)
+            # Sets file's current position at offset.
+            fp.seek(0)
+            # convert back to json.
+            json.dump(file_data, fp)
+
+'''
+# For saving into the database with one table
+'''
+def write_json(new_item):
```

Comment:
maybe the function name should not be `write_json` as it's not writing any json

Reply:
> maybe the function name should not be `write_json` as it's not writing any json

I just realised I forgot to change the function name. Thank you for the comment!